.gitlab-ci.yml

```diff
@@ -50,6 +51,42 @@ stages:
 ###############
 # stage build #
 ###############
+prepare:squidex:branch:
+  <<: *tmpl_branch
+  image: $INTEGRATION_DOCKER_IMAGE
+  stage: build
+  environment:
+    name: review/$CI_EXTERNAL_PULL_REQUEST_IID
+    url: https://$CI_EXTERNAL_PULL_REQUEST_IID.$ASAP_HOSTNAME
+    auto_stop_in: 3 days
```

Comment:
whats this for? i dont think we have a `stop` action that corresponds to this? (the env teardown is in github actions)

unless it has another function?